calico/reference/legal/typha.md

```diff
@@ -1477,7 +1477,7 @@ canonical_url: '/reference/legal/typha'
 		the same "printed page" as the copyright notice for easier identification within
 		third-party archives.
 
-		   Copyright 2014 Unknwon
+		   Copyright 2014 Unknown
```

Comment:
I'm curious if this is a real typo or should some proper name be placed in here.

Reply:
FWIW i don't see any authors named anything like Unknwon in `git log`

Reply:
That may not be a sufficient search. Looks like calico was moved around a bit, and broken out from a monorepo, and commit history would have been lost during that. I'll just revert anything legal related.